scripts/cnv_wdl/somatic/README.md

```diff
@@ -0,0 +1,58 @@
+## Running the Somatic CNV WDL
```

Comment:
Yes, it's different.  I only kept descriptions of the workflow-level parameters and examples of a few task-level parameters.